qlib/backtest/report.py

```diff
@@ -357,9 +358,11 @@ def _get_base_vol_pri(
         agg = pa_config.get("agg", "twap").lower()
         price = pa_config.get("price", "deal_price").lower()
 
-        # NOTE:  IndexTradeRange is not supported!!!!! Because inner index is not available
-        trade_start_time, trade_end_time = decision.trade_range.clip_time_range(
-            start_time=trade_start_time, end_time=trade_end_time
+        if(decision.trade_range is not None):
```

Comment:
trade_range is not an attribute of  BaseTradeDecision.

Don't use the attribute of the subclass